runtime/doc/lua.txt

```diff
@@ -2970,6 +2979,30 @@ vim.fs.normalize({path}, {opts})                          *vim.fs.normalize()*
 
         vim.fs.normalize('$XDG_CONFIG_HOME/nvim/init.vim')
         -- '/Users/jdoe/.config/nvim/init.vim'
+
+        vim.fs.normalize('~/src/nvim/api/../tui/./tui.c')
+        -- '/home/jdoe/src/nvim/tui/tui.c'
+
+        vim.fs.normalize('.')
+        -- '.'
+
+        vim.fs.normalize('./foo/bar')
+        -- 'foo/bar'
+
+        vim.fs.normalize('foo/../../../bar')
+        -- '../../bar'
+
+        vim.fs.normalize('/home/jdoe/../../../bar')
+        -- /bar
+
+        vim.fs.normalize('C:foo/../../baz')
+        -- 'C:../baz'
+
+        vim.fs.normalize('C:/foo/../../baz')
+        -- C:/baz
+
+        vim.fs.normalize('\\\\?\\UNC\\server\\share\\foo\\..\\..\\..\\bar')
```

Comment:
use `[[...]]`, the escaping is just distracting from the purpose of this example.